File: go/SequentialSearch.go
package main

import (
	"math/rand"
	"fmt"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())
	length := rand.Intn(10) + 10
	arr := make([]int, length)
	for i := 0; i < len(arr); i++ {
		arr[i] = rand.Intn(100)
	}
	fmt.Println(arr)
	val := arr[rand.Intn(len(arr))]
	fmt.Println(SequentialSearch(arr, val))
}

func SequentialSearch(arr []int, val int) (int, int) {
	for i := 0; i < len(arr); i++ {
		if arr[i] == val {
			return i, val
		}
	}
	return 0, 0
}

File: python/SequentialSearch.py
import random


def SequentialSearch(arr, val):
    i = 0
    while i < len(arr):
        if arr[i] == val:
            return i, val
        i += 1


arr = [0] * random.randint(10, 20)
i = 0
while i < len(arr):
    arr[i] = random.randint(0, 100)
    i += 1
val = arr[random.randint(0, len(arr))]
print(arr)
print(SequentialSearch(arr, val))
